bubble_sort.rb

```diff
@@ -0,0 +1,28 @@
+# bubble_sort
+
+def bubble_sort(arr)
+  count = arr.length - 1
+  until count.zero?
+    count.times do |i|
+      arr[i], arr[i + 1] = arr[i + 1], arr[i] if arr[i] > arr[i + 1]
+    end
+    count -= 1
+  end
+  arr
+end
+
+bubble_sort([99, 88, 77, 66, 55, 44, 33, 22, 11, 0])

```

Comment:
There should be no debugging instructions in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)